llvm-external-projects/iree-dialects/test/lit.cfg.py

```diff
@@ -65,6 +65,12 @@
 tool_dirs = [config.llvm_tools_dir]
 tools = [
     ToolSubst('%PYTHON', config.python_executable, unresolved='ignore'),
+    # Since we build iree-dialects out of tree, we don't have a common tools
+    # directory, so substitute binaries needed to an explicit path.
+    ToolSubst(
+        'iree-dialects-opt',
+        os.path.join(config.iree_dialects_obj_root,
+                     'tools/iree-dialects-opt/iree-dialects-opt'))
```

Comment:
A few more CI failures with `FileCheck: command not found` (why is only the ASan job failing...?):

https://github.com/iree-org/iree/actions/runs/4199948698/jobs/7285451972
```
[51/52] Running the iree-dialects regression tests
-- Testing: 33 tests, 33 workers --
Testing:  0.. 10
FAIL: IREE_DIALECTS :: Dialect/linalg_transform/scoped.mlir (6 of 33)
******************** TEST 'IREE_DIALECTS :: Dialect/linalg_transform/scoped.mlir' FAILED ********************
Script:
--
: 'RUN: at line 1';   /work/build-asan/llvm-external-projects/mlir-iree-dialects/tools/iree-dialects-opt/iree-dialects-opt --test-wrap-scope='opname=arith.addi' /work/llvm-external-projects/iree-dialects/test/Dialect/linalg_transform/scoped.mlir | FileCheck /work/llvm-external-projects/iree-dialects/test/Dialect/linalg_transform/scoped.mlir --check-prefix WRAP
: 'RUN: at line 2';   /work/build-asan/llvm-external-projects/mlir-iree-dialects/tools/iree-dialects-opt/iree-dialects-opt --test-unwrap-scope /work/llvm-external-projects/iree-dialects/test/Dialect/linalg_transform/scoped.mlir | FileCheck /work/llvm-external-projects/iree-dialects/test/Dialect/linalg_transform/scoped.mlir --check-prefix UNWRAP
--
Exit Code: 127

Command Output (stderr):
--
/work/build-asan/llvm-external-projects/mlir-iree-dialects/test/Dialect/linalg_transform/Output/scoped.mlir.script: line 1: FileCheck: command not found
```